src/ui/Resizable/resizable.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+
+import * as ResizablePrimitive from "react-resizable-panels";
+import classes from "./Resizable.module.css";
+
+const GripVertical = (props: React.ComponentProps<"svg">) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width={24}
+    height={24}
+    viewBox="0 0 24 24"
+    fill="none"
+    stroke="currentColor"
+    strokeWidth={2}
+    strokeLinecap="round"
+    strokeLinejoin="round"
+    className="lucide lucide-grip-vertical"
+    {...props}
+  >
+    <title>Grip Vertical Icon</title>
+    <circle cx={9} cy={12} r={1} />
+    <circle cx={9} cy={5} r={1} />
+    <circle cx={9} cy={19} r={1} />
+    <circle cx={15} cy={12} r={1} />
+    <circle cx={15} cy={5} r={1} />
+    <circle cx={15} cy={19} r={1} />
+  </svg>
+);
```

Comment:
this is not needed. If you'd have inspected the package.json, you'd have noticed that this project uses react-icons.

So you can use:

import { RxDragHandleDots2 } from "react-icons/rx";
<RxDragHandleDots2 />